components/MobileMenu/MobileMenu.tsx

```diff
@@ -0,0 +1,30 @@
+
+import React from 'react'
```

Comment:
Don't need to import react

Reply:
For some reason when I run it on my laptop it wasnt working saying react not found unless i put it there were defect in that laptop it should still run after removing those